app/services/prison_visits/client.rb

```diff
@@ -2,9 +2,15 @@
 
 module PrisonVisits
   class Client
+    TIMEOUT = 2 # seconds
+
     def initialize(host)
       @host = host
-      @connection = Excon.new(host, persistent: true)
+      @connection = Excon.new(host,
+                              persistent: true,
+                              connect_timeout: TIMEOUT,
+                              read_timeout: TIMEOUT,
+                              write_timeout: TIMEOUT)
```

Comment:
Can I be a real pedant and object to this formatting? If you put `host` on a new line, rubocop won't push everything out to the right, and it will be much nicer, and be consistent with the rest of the codebase.